go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/felixge/httpsnoop v1.0.4
 	github.com/ipfs-shipyard/nopfs v0.0.12
 	github.com/ipfs-shipyard/nopfs/ipfs v0.13.2-0.20231024163508-120e0c51ee3a
-	github.com/ipfs/boxo v0.19.1-0.20240426120703-0f223aada9b8
+	github.com/ipfs/boxo v0.19.1-0.20240523201844-5dcc928943b6
```

Comment:
TODO: update

Reply:
- [ ]  @lidel will rebase on top of #150